<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GL Trace Statistics</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>GL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GLX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WGL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CGL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>total</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>per frame</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>11</number>
        </property>
        <widget class="QWidget" name="tabGlCalls">
         <attribute name="title">
          <string>GL</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableView" name="tvGlCalls"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlCallsPf">
         <attribute name="title">
          <string>GL</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableView" name="tvGlCallsPf"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlExt">
         <attribute name="title">
          <string>GL Ext</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableView" name="tvGlExt"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlExtPf">
         <attribute name="title">
          <string>GL Ext</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTableView" name="tvGlExtPf"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlxCalls">
         <attribute name="title">
          <string>GLX</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTableView" name="tvGlxCalls"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlxCallsPf">
         <attribute name="title">
          <string>GLX </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTableView" name="tvGlxCallsPf"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWglCalls">
         <attribute name="title">
          <string>WGL</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QTableView" name="tvWglCalls"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWglExt">
         <attribute name="title">
          <string>WGL Ext</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QTableView" name="tvWglExt"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWglExtPf">
         <attribute name="title">
          <string>WGL Ext</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QTableView" name="tvWglExtPf"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWglCallsPf">
         <attribute name="title">
          <string>WGL</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QTableView" name="tvWglCallsPf"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlxExt">
         <attribute name="title">
          <string>GLX Ext</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QTableView" name="tvGlxExt"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabGlxExtPf">
         <attribute name="title">
          <string>GLX Ext</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QTableView" name="tvGlxExtPf"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
